Dockerfile

```
@@ -1,15 +1,32 @@
 
-RUN pip install .
+WORKDIR /app
 
-RUN curl -L -o helm-docs.deb https://github.com/norwoodj/helm-docs/releases/download/v${HELM_DOCS_VERSION}/helm-docs_${HELM_DOCS_VERSION}_Linux_x86_64.deb \
- && apt install ./helm-docs.deb \
- && rm -f ./helm-docs.deb
+# Copy and install dependencies first for better caching
+COPY pyproject.toml poetry.lock ./
 
-ENTRYPOINT ["python", "/usr/local/bin/helm-charts-updater"]
+RUN pip install --no-cache-dir poetry && \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)